libs/editor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergContainerFragment.java

```diff
@@ -170,6 +174,14 @@ public void onPause() {
     public void onResume() {
         super.onResume();
 
+        OnBackPressedCallback callback = new OnBackPressedCallback(true) {
+            @Override
+            public void handleOnBackPressed() {
+                mWPAndroidGlueCode.onBackPressed();
+            }
+        };
+
+        getActivity().getOnBackPressedDispatcher().addCallback(this, callback);
```

Comment:
I thought we might need to remove the callback when the fragment is destroyed, however, seems this is already covered by Android ([reference](https://developer.android.com/reference/androidx/activity/OnBackPressedDispatcher#addCallback(androidx.lifecycle.LifecycleOwner,androidx.activity.OnBackPressedCallback))).

Reply:
Yup I thought that too but by looking at other places in the app confirmed it wasn't needed as well as the docs reference you mentioned 😄